.changes/nextrelease/bugfix-unicode-characters-multipartcopy

```diff
@@ -0,0 +1,7 @@
+[
+    {
+        "type": "bugfix",
+        "category": "MultipartCopy",
```

Comment:
Category refers to the namespace, so in this case it should be `S3`.

Reply:
Thank you, that has been updated